terraso_backend/apps/auth/views.py

```diff
@@ -139,12 +137,25 @@ def post(self, request, *args, **kwargs):
             )
             return JsonResponse({"error": "User not found"}, status=400)
 
-        access_token = jwt_service.create_access_token(user)
-        refresh_token = jwt_service.create_refresh_token(user)
+        access_token, refresh_token = terraso_login(self.request, user)
 
         return JsonResponse(
             {
                 "access_token": access_token,
                 "refresh_token": refresh_token,
             }
         )
+
+
+class LogoutView(View):
+    def post(self, request, *args, **kwargs):
+        dj_logout(request)
+        return HttpResponse("OK", status=200)
```

Comment:
Question: Will this be displayed to the user, or is the intent that a client application will call this and show the proper message to the user?

Reply:
The intent is that a client application will call this and show the proper message to the user.